advisories/github-reviewed/2022/06/GHSA-jq66-xh47-j9f3/GHSA-jq66-xh47-j9f3.json

```diff
@@ -15,35 +15,11 @@
     }
   ],
   "affected": [
-    {
-      "package": {
-        "ecosystem": "crates.io",
-        "name": "streebog"
-      },
-      "ranges": [
-        {
-          "type": "ECOSYSTEM",
-          "events": [
-            {
-              "introduced": "0"
-            },
-            {
-              "fixed": "0.8.0"
-            }
-          ]
-        }
-      ]
-    },
     {
       "package": {
         "ecosystem": "crates.io",
         "name": "failure"
       },
-      "ecosystem_specific": {
-        "affected_functions": [
-          "failure::Fail::__private_get_type_id__"
-        ]
-      },
```

Comment:
Hm, I didn't mean for this to get deleted. Seems to have gotten removed by the suggest improvements interface.

Reply:
All good. That's a bug we need to get handled. I'll make sure that doesn't propagate 😄